_main.cfg

```
@@ -58,17 +58,19 @@ data/add-ons/Flight_Freedom_Resources/music#enddef
     {ENABLE_DWARVISH_RUNESMITH}
     {ENABLE_TROLL_SHAMAN}
     {ENABLE_SAURIAN_SPEARTHROWER}
+    {ENABLE_NIGHTBLADE}
+    {ENABLE_PARAGON}
 
     [modify_unit_type]
         type=Drake Clasher
         add_advancement=Drake Guard
-        set_experience=41
+        #set_experience=41
     [/modify_unit_type]
 
     [modify_unit_type]
         type=Naga Fighter
         add_advancement=Naga Depthstalker
-        set_experience=33
+        #set_experience=33
```

likewise, why comment this out?

---

As with my previous suggestion:
```suggestion
#ifdef FORCE_EXP_AMT
        set_experience=33
#endif
```

---

fixed as of f398700.